### DISCUSSION TITLE
Recurring Events - Additional Parameters

### DISCUSSION THREAD

**Discussion by valiantgenomics** [davish/obsidian-full-calendar#452] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

Another idea, if I may, in relation to [Events - Copy + Paste #450](https://github.com/davish/obsidian-full-calendar/discussions/450)

Would it be possible to add an additional parameter for the recurring events, where you can set them to repeat on the 1st, 2nd, 3rd, 4th, 5th, etc. of every month?

This allows for much more granularity. Because currently we can only segment them by days of the week - Mon, Tue, Wed, etc.

But for events that repeat on the 1st of every month, for example, we can't do anything about and have to keep re-creating them.

Thanks again for your consideration!

### ✔️ Solution

Add an additional parameter for the recurring events, where you can set them to repeat on the 1st, 2nd, 3rd, 4th, 5th, 6th, 7th, 8th, 9th, 10th, etc. of every month.

**Comment by Grant837** [comment]
Of course, more patterns will be needed.  I would like ´every other xxx´ , in particular, every other week...
